packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -88,6 +95,7 @@ export default class QuanticSearchInterface extends LightningElement {
       );
       if (!redirectData) {
         engine.executeFirstSearch();
+        this.isInitialized = true;
```

Comment:
Can we do this instead of having 2 `this.isInitialized = true;` ?

```suggestion
/**
 * @param {import("coveo").SearchEngine} engine
 */
initialize = (engine) => {
  const { updateQuery } = CoveoHeadless.loadQueryActions(engine);
  if (!this.disableStateInUrl) {
    this.initUrlManager(engine);
  }
  if (!this.skipFirstSearch) {
    const redirectData = window.localStorage.getItem(
      STANDALONE_SEARCH_BOX_STORAGE_KEY
    );
    if(redirectData) {
      window.localStorage.removeItem(STANDALONE_SEARCH_BOX_STORAGE_KEY);
      const { value, analytics } = JSON.parse(redirectData);
      engine.dispatch(updateQuery({ q: value }));
      engine.executeFirstSearchAfterStandaloneSearchBoxRedirect(analytics);
    } else {
      engine.executeFirstSearch();
    }
  }
  this.isInitialized = true;
};
```

Reply:
Yes, that's better